shuffle-plugin/src/main/scala/com/nvidia/spark/rapids/shuffle/ucx/UCXTransaction.scala

```diff
@@ -403,32 +319,35 @@ private[ucx] class UCXTransaction(conn: UCXConnection, val txId: Long)
       errorMessage = Option(errorMsg))
   }
 
-  def completeCancelled(requestType: RequestType.Value, hdr: Long): Unit = {
+  def completeCancelled(messageType: MessageType.Value, hdr: Long): Unit = {
     complete(TransactionStatus.Cancelled,
-      messageType = Option(requestType),
+      messageType = Option(messageType),
       header = Option(hdr))
   }
 
-  def completeWithSuccess(
-    messageType: RequestType.Value,
-    hdr: Option[Long],
-    message: Option[RefCountedDirectByteBuffer]): Unit = {
+  def completeWithSuccess(messageType: MessageType.Value,
+      hdr: Option[Long],
+      message: Option[TransportBuffer]): Unit = {
     complete(TransactionStatus.Success,
       messageType = Option(messageType),
       header = hdr,
       message = message)
   }
 
   // Reference count is not updated here. The caller is responsible to close
-  private[ucx] def setActiveMessageData(data: Option[RefCountedDirectByteBuffer]): Unit = {
+  private[ucx] def setActiveMessageData(data: Option[TransportBuffer]): Unit = {
     activeMessageData = data
   }
 
   // Reference count is not updated here. The caller is responsible to close
-  override def releaseMessage(): RefCountedDirectByteBuffer = {
+  override def releaseMessage: MetadataTransportBuffer = {
```

Comment:
Curious why parentheses are removed on this method?  Typically omitting parentheses on a method should only be done for accessor-like methods that don't mutate the instance state.